pkg/api/const.go

```diff
@@ -288,6 +288,14 @@ const (
 	DefaultImageVersion = "17763.737.1909062324"
 )
 
+// Aks windows VHD parts
+const (
```

Comment:
Is there any practical way to converge where we maintain Windows VHD default references and where we maintain the Linux defaults? See here for Linux VHD config:

https://github.com/Azure/aks-engine/blob/master/pkg/api/azenvtypes.go#L104

Reply:
Hmm....  we could add now image config types there and then just grab the values we need installed of looking them up in a 'disto map'.
That wouldn't change the behavior in code but would keep all the image related changes to a single file.
Let me give that a try.

Reply:
Yeah I assume things are functionally ready here, just observing that we shouldn't be triaging Windows from Linux coding interfaces if we can help it. (A good justification for doing so would be that the existing Linux implementation has lots of tech debt and the Windows surface area is an excuse to improve maintenance. :))

Reply:
Great sell!

Reply:
@jackfrancis can you take a look at the lastest commit? I feel it is too big of a change to use how windows images are refenced internal to aks-engine but i did move the defaults align how defaults are specified into azenvtypes.go to align with how linux images are specified.